samples/namespacescope-sample/createUIDefinition.json

```diff
@@ -0,0 +1,247 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+	"handler": "Microsoft.Azure.CreateUIDef",
+	"version": "0.1.2-preview",
+	"parameters": {
+		"config": {
+			"isWizard": false,
+			"basics": {
+				"location": {
+					"visible": "[basics('createNewCluster')]",
+					"allowedValues": [
+						"eastus",
```

Comment:
Let's update this to have the current supported region?

Reply:
here is the regions list:

East US
 West US
 Central US
 West Central US
 South Central US
 East US 2
 West US 2
 West Europe
 North Europe
 Canada Central
 Southeast Asia
 Australia East
 Central India

Reply:
will do as a separate PR. we may need to educate on the geography mapping to the regions too